[Bug]: [Nightly]Insert failed reporting 'AssertionError: Regex pattern did not match.'

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:a8e2a69
- Deployment mode(standalone or cluster):cluster
- SDK version(e.g. pymilvus v2.0.0rc2):2.2.0.dev68
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```


### Current Behavior

Insert failed reporting 'AssertionError: Regex pattern did not match.'
```
[2022-10-30T19:22:55.837Z] >           future.result()
[2022-10-30T19:22:55.837Z] E           AssertionError: Regex pattern did not match.
[2022-10-30T19:22:55.837Z] E            Regex: 'partitionID of partitionName:p can not be find'
[2022-10-30T19:22:55.837Z] E            Input: '<MilvusException: (code=1, message=partition is not exist: p)>'
[2022-10-30T19:22:55.837Z] 
```

### Expected Behavior

PASS

### Steps To Reproduce

```markdown
https://jenkins.milvus.io:18080/blue/organizations/jenkins/Milvus%20Nightly%20CI/detail/master/178/pipeline/155/
```


### Milvus Log

_No response_

### Anything else?

_No response_

--- a/tests/python_client/testcases/test_insert.py
+++ b/tests/python_client/testcases/test_insert.py
@@ -913,7 +913,7 @@ class TestInsertAsync(TestcaseBase):
         """
         collection_w = self.init_collection_wrap(name=cf.gen_unique_str(prefix))
         df = cf.gen_default_dataframe_data()
-        err_msg = "partitionID of partitionName:p can not be find"
+        err_msg = "partition is not exist: p"
         future, _ = collection_w.insert(data=df, partition_name="p", _async=True)
         future.done()
         with pytest.raises(MilvusException, match=err_msg):

